### PR TITLE
Quest fixes and rewordings

### PR DESCRIPTION
**Pull request by chill-was-taken**

### CODE REVIEW

File: config/betterquesting/DefaultQuests/Quests/MultipleQuestLine/BetterQuesting-AAAAAAAAAAAAAAAAAAAF3A==.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
...
...to sync the quests with someone in your party, then use this. They won\u0027t get any rewards for that though.\n\n/bq_admin hardcore§r\nEnables BQ3 Hardcore Mode. You only start with 3 lives, so best of luck. Additional life hearts are available in the Kill all the Things tab.\n\nAside from the consume-type quests, a quest counts for everyone in the party if one person finished it. However, if someone joined your party later, you need to get them up to speed first either through /bq_copyquests above, or manually. Otherwise, your quest completion won\u0027t count for them because the requirements for that quest are not fulfilled for them.\n\nIf your singleplayer world does not have cheats enabled, go to the ESC screen, open to LAN, then enable cheats. This makes pausing impossible until you reload afterwards. If you host a server, you can run the commands in the command line, without \"/\". You can also edit the level.dat file, and set \u0027allowCommands\u0027 to 1, but make sure to backup your world before doing that.\n\n§4For quests to save properly, you must be online and logged into your Mojang account when starting the game. Sorry, it\u0027s a limitation of Better Questing.§r"
```
Comment (chochem) [inline config/betterquesting/DefaultQuests/Quests/MultipleQuestLine/BetterQuesting-AAAAAAAAAAAAAAAAAAAF3A==.json:34]: edit: I wouldnt recommend edit mode to players to 'view all quests and their requirements'. That is not what its for and will just lead to them not being able to complete quests anymore. just say its for editting quests instead.

save: does not save edit mode or progress. just remove that comment.

**Review by chochem — DISMISSED** [review]
I have to say I am not a fan of replacing all the LV/MV/etc by Low Voltage(LV) and Medium Voltage(MV) and so on. Nobody says that and the Questlines are called LV/MV/... too. It's just lengthy and awkward.

**Comment by chill-was-taken** [comment]
My intention was not to replace them everywhere. Rather to replace it once or twice to give an idea what it is about. For instance, the Creosote quest previously said "RC boiler". A player has no idea what an RC boiler is at that point unless they played TiC before.

File: config/betterquesting/DefaultQuests/Quests/MultipleQuestLine/BetterQuesting-AAAAAAAAAAAAAAAAAAAF3A==.json (hunk 1 above)
Comment (chill-was-taken) [reply config/betterquesting/DefaultQuests/Quests/MultipleQuestLine/BetterQuesting-AAAAAAAAAAAAAAAAAAAF3A==.json:34]: ok, got it, working on it

File: config/betterquesting/DefaultQuests/Quests/MultipleQuestLine/ETOOMANYKEYS-AAAAAAAAAAAAAAAAAAAIYQ==.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
...
...many keys to fit a 104 key keyboard.\n\nThere are lots of keybinds in this modpack, to avoid conflicts most of them are unbinded by default and only those available from the start of the game are binded.\nHere is a list of what some of them do :\n\n\nBetterQuesting\nGRAVE - Open Questbook\n\nBattlegear\nR - Draw Weapons - It\u0027s a second hotbar that holds weapons\n\nDraconic Evolution\nV - Place the item you are holding on the block you are looking at\n\nJourneymap\nB - Create a Waypoint\nJ - Open Fullscreen Map\n\u003d - Zoom In (works on minimap too)\n- - Zoom out\n\nInventory Tweaks\nR - Sort Inventory\n\nShare Where I am\nINSERT - Places a temporary beam where you are looking at that will be seen by you and other players\nSHIFT + INSERT - Places a temporary beam where §oyou§r are that will be seen by you and other players\nCTRL + INSERT - Shares the position you are looking at with other players\nCTRL + SHIFT + INSERT - Shares your position with other players\n\nWaila - Shows information about what you are looking at\nNUMPAD1 - show/hide the Waila HUD\nNUMPAD3 - shows how to craft what you are looking at\nNUMPAD4 - shows the crafting usages of the thing you are looking at\n"
```
Comment (chochem) [inline config/betterquesting/DefaultQuests/Quests/MultipleQuestLine/ETOOMANYKEYS-AAAAAAAAAAAAAAAAAAAIYQ==.json:34]: looks like you reverted your last fix here. also explains the merge conflict

**Comment by chochem** [comment]
yea once or twice is fine I guess

File: config/betterquesting/DefaultQuests/Quests/MultipleQuestLine/ETOOMANYKEYS-AAAAAAAAAAAAAAAAAAAIYQ==.json (hunk 2 above)
Comment (chill-was-taken) [reply config/betterquesting/DefaultQuests/Quests/MultipleQuestLine/ETOOMANYKEYS-AAAAAAAAAAAAAAAAAAAIYQ==.json:34]: Yes, I resolved this conflict on local, haven't pushed yet. Working on your suggestions.

**Comment by chill-was-taken** [comment]
ok what just happened with that push, lemme think

**Comment by chill-was-taken** [comment]
Did `git rebase` change the other commit hashes because I rebased too deep?

**Comment by chill-was-taken** [comment]
When I look at the diff in a new PR, it looks okay, without the files that I didn't change. Ok, I'll make a new one.

**Pull request closed without merging**